### PR TITLE
Add headers1A property to the AST for both API Blueprint and Refract adapters

### PR DESCRIPTION
**Pull request by michalholasek**
Changes made in this PR are needed for support of multiple headers (with the same key / header name) in the core app.

### CODE REVIEW

**Comment by michalholasek** [comment]
[Console fails to render header](https://trello.com/c/XQZXbl8P/279-console-fails-to-render-header)

**Comment by pksunkara** [comment]
Looks good, but please add minor test for this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Metamorphoses Changelog
 
+## 0.13.8
+
+### Enhancements
+
+- `headers1A` property added to the Request / Response transaction pairs for both API Blueprint and Refract adapters.
```
Comment (XVincentX) [inline CHANGELOG.md]: Please explain why we made that (not losing the headers)

File: src/adapters/refract/transformTransactions.coffee [hunk 2]
```diff
@@ -65,11 +65,12 @@ module.exports = (transactions, options) ->
 
     # Check for empty http request
     requestName = _.chain(httpRequest).get('meta.title', '').contentOrValue().value()
-    requestHeaders = getHeaders(httpRequest)
     requestBody = trimLastNewline(if _.content(httpRequestBody) then _.content(httpRequestBody) else '')
     requestSchema = trimLastNewline(if _.content(httpRequestBodySchemas) then _.content(httpRequestBodySchemas) else '')
     requestAuthSchemes = transformAuth(httpTransaction, options)
 
+    [requestHeaders, requestHeaders1A] = getHeaders(httpRequest)
```
Comment (XVincentX) [inline src/adapters/refract/transformTransactions.coffee]: I think we should return an object rather than an array - it's confusing.

**Review by XVincentX — CHANGES_REQUESTED** [review]
Please add a test for checking that when you're putting two headers, those get persisted in the new property.

File: src/adapters/refract/transformTransactions.coffee (hunk 2 above)
Comment (pksunkara) [reply src/adapters/refract/transformTransactions.coffee]: Actually, `getHeaders` should probably only return `headers` and we can add `getHeaders1A` to return `headers1A`

Comment (XVincentX) [reply src/adapters/refract/transformTransactions.coffee]: You decide on that, let's just not use arrays as a returning pattern as long it's not really same type.

Comment (michalholasek) [reply src/adapters/refract/transformTransactions.coffee]: I agree that returning arrays with two (or more) different data structures is not very elegant and good for readibility, but I wanted to keep it DRY. I will add `getHeaders1A`.

File: CHANGELOG.md (hunk 1 above)
Comment (michalholasek) [reply CHANGELOG.md]: Addressed.

File: src/adapters/refract/transformTransactions.coffee (hunk 2 above)
Comment (michalholasek) [reply src/adapters/refract/transformTransactions.coffee]: Addressed.

**Review by pksunkara — APPROVED** [review]

File: test/transformation-test.coffee [hunk 3]
```diff
@@ -160,6 +166,15 @@ describe('Transformations', ->
...
               assert.equal(ast.sections[0].resources[0].responses[0].body, 'Hello World')
           )
+          it('response has headers1A property', ->
+            assert.isDefined(ast.sections[0].resources[0].responses[0].headers1A)
+          )
+          it('response has headers1A with three Set-Cookie headers', ->
+            setCookieHeaders = ast.sections[0].resources[0].responses[0].headers1A.filter((item) ->
+              item.name is 'Set-Cookie'
+            )
+            assert.equal(setCookieHeaders.length, 3)
```
Comment (XVincentX) [inline test/transformation-test.coffee]: Could you please check that the value is different for all these? (Make sure it's not being overwritten)

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/metamorphoses",
-  "version": "0.13.7",
+  "version": "0.13.8",
```
Comment (XVincentX) [inline package.json:3]: This is a minor release, not a patch. It's adding a new property.

**Review by XVincentX — CHANGES_REQUESTED** [review]
1. Bump to 0.14
2. Add a test making sure the value is different.

File: package.json (hunk 4 above)
Comment (pksunkara) [reply package.json:3]: From semver

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

When less than v1, generally, people follow the rule of doing minor changes as patch versions and major changes as minor versions.

Comment (XVincentX) [reply package.json:3]: Ah - ok.

File: test/transformation-test.coffee (hunk 3 above)
Comment (michalholasek) [reply test/transformation-test.coffee]: Addressed.

**Review by XVincentX — APPROVED** [review]

**Comment by XVincentX** [comment]
I think we can move ahead!

**Pull request merged**